### PR TITLE
further reduce api response time by indexing DB tables that can be filtered via in /explorer

### PR DESCRIPTION
**Pull request by owocki**
##### Description

further reduce api speed by indexing DB tables that can be filtered via in /explorer

takes the API reponse time down from 0.45s to 0.2s in some situations

##### Refers/Fixes

https://github.com/gitcoinco/web/pull/4550

##### Testing

tested locally

### CODE REVIEW

**Comment by owocki** [comment]
@danlipert i suppose we could `index_together` a lot of these fields, but there are many many many permutations here.. not sure it's worth it.

**Comment by owocki** [comment]
f78940807f561254c4a86700410a77ca82661541 adds some index_togethers!

**Comment by kuhnchris** [comment]
you mean "increase API speed" or maybe "decrease response time" or "increase responsiveness"?

File: app/dashboard/models.py [hunk 1]
```diff
@@ -273,13 +290,13 @@ class Bounty(SuperModel):
...
-    project_type = models.CharField(max_length=50, choices=PROJECT_TYPES, default='traditional')
-    permission_type = models.CharField(max_length=50, choices=PERMISSION_TYPES, default='permissionless')
+    project_type = models.CharField(max_length=50, choices=PROJECT_TYPES, default='traditional', db_index=True)
+    permission_type = models.CharField(max_length=50, choices=PERMISSION_TYPES, default='permissionless', db_index=True)
     repo_type = models.CharField(max_length=50, choices=REPO_TYPES, default='public')
     snooze_warnings_for_days = models.IntegerField(default=0)
     is_featured = models.BooleanField(
         default=False, help_text=_('Whether this bounty is featured'))
-    featuring_date = models.DateTimeField(blank=True, null=True)
+    featuring_date = models.DateTimeField(blank=True, null=True, db_index=True)
```
Comment (thelostone-mc) [inline app/dashboard/models.py]: @owocki / @danlipert do we need to index  `featuring_date`?

Comment (owocki) [reply app/dashboard/models.py]: what do we need to fix about it?

> indexing == yay but only if it's legit needed right ?

 IMHO the trade off here is pretty positive ROI.  indexing on a table the size of `dashboard_bounty` is cheap (in terms of storage space + write speed) + it improves lookup times quite significantly

**Comment by owocki** [comment]
do i press the big scary merge button?  or do u guys do that?

**Comment by thelostone-mc** [comment]
@owocki we'll get it in :P

**Comment by danlipert** [comment]
@octavioamu you want to take a look at this? Also @thelostone-mc we should rebase and merge the migrations

**Comment by thelostone-mc** [comment]
Closing this out and getting https://github.com/gitcoinco/web/pull/4812 in instead

**Pull request closed without merging**